### ISSUE TITLE
make the OAF libraries browsable in practice

### ISSUE THREAD

**Issue by kohlhase** [MathHubInfo/Frontend#68]
I still have great trouble demoing OAF, since the formal libraries are not browsable 
- Mizar (https://mathhub.info/Mizar/MML) gives me an almost un-readable presentation where all links and buttons do not work. The only archive MML does not have a `desc.html` or title. 
- HOL light (https://mathhub.info/HOLLight/basic) shows lots of subdirs and theory files, bust most links (e.g. https://mathhub.info/HOLLight/basic/calc_num.omdoc) are dangling.  
- TPS (https://mathhub.info/tps/tps) is empty (I can understand that) but leads to a drupal error
- MMT (https://mathhub.info/MMT) works best of these, but already https://mathhub.info/MMT/LATIN/foundations shows a filed conversion and nothing else. 
  Also, we should really reorganize MMT, so the LATIN becomes its own library (I think).
  @florian-rabe @cmaeder @tkw1536

**Comment by tkw1536** [comment]
We should add all of these to MathHub. Not sure why we haven't enabled them yet.

**Comment by Jazzpirate** [comment]
TPS doesn't exist, but you're missing (at least?) Coq, PVS and IMPS, I guess...?

**Comment by kohlhase** [comment]
TPS used to exist, it has to be somewhere.

**Comment by Jazzpirate** [comment]
I'm pretty sure it doesn't ;) I remember you once giving me the OMDoc and wanting me to write an importer (that was years ago in Bremen), at which I failed because I was a) busy with other things and b) incapable of understanding where the URIs point to, and hence, resolving them to anything meaningful.

Unless someone else has done it in the meantime, I'm pretty sure that is still the state of TPS.

**Comment by kohlhase** [comment]
We seem to have a different notion of existence, whereas for you that means "readable by MMT" I am less discerning.

**Comment by Jazzpirate** [comment]
well, if it's not readable by MMT, I'm pretty sure MathHub can't do anything with it...?

**Comment by kohlhase** [comment]
In any case, we need at least one more library to have a credible MWS search paper.

**Comment by Jazzpirate** [comment]
What about the others I mentioned? PVS, Coq, IMPS... PVS might be a good target since that one should "work" and not be as overwhelmingly large as Coq, and IMPS... I don't know, we'd have to ask Jonas

**Comment by kohlhase** [comment]
Yes, PVS would be a very good target, especially, since we know that the MMT is OK.

**Comment by Jazzpirate** [comment]
(or at least *should* be)

**Comment by tkw1536** [comment]
I'll try and load PVS today then and see what happens.
